src/model/Model.lua

```diff
@@ -16,7 +16,6 @@ local Config = require("src/config/Config")
 local Policy = require("src/model/Policy")
 local Assertion = require("src/model/Assertion")
 local Util = require("src/util/Util")
-
```

Comment:
Please don't remove this line since this is meant to separate `Model` from loaded modules.